e2etest/bigint_test.go

```diff
@@ -88,16 +88,18 @@ func prepareBlockchain(ctx context.Context, _executor string, r *require.Asserti
 	r.NoError(rp.Register(registry))
 	sf, err := factory.NewFactory(cfg, factory.InMemTrieOption(), factory.RegistryOption(registry))
 	r.NoError(err)
+	genericValidator := protocol.NewGenericValidator(sf, accountutil.AccountState)
 	ap, err := actpool.NewActPool(sf, cfg.ActPool)
 	r.NoError(err)
+	ap.AddActionEnvelopeValidators(genericValidator)
```

Comment:
why add this? the test (as of now) is not properly written?

Reply:
the current code is not written properly